packages/docs/package.json

```diff
@@ -26,7 +26,7 @@
   },
   "dependencies": {
     "clsx": "2.1.1",
-    "next": "14.2.25",
+    "next": "14.2.26",
```

Comment:
Are there any breaking changes for docs in next15? If not we could update to latest next.